src/components/TempoDropdown.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import {
+  Select,
+  SelectOption,
+  SelectVariant,
+  SelectOptionProps,
+} from '@patternfly/react-core';
+
+type TempoDropdownProps = {
+  id: string;
+  selectionOptions: SelectOptionProps[];
+  selectedTempoList: string | undefined;
+  selectedNamespace: string | undefined;
+  setTempoList: (
+    selectedTempoStack?: string,
+    selectedNamespace?: string,
+  ) => void;
+};
+
+export const TempoDropdown = (props: TempoDropdownProps) => {
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState<string>(
+    props.selectedTempoList && props.selectedNamespace
+      ? props.selectedNamespace + ' / ' + props.selectedTempoList
+      : '',
+  );
+
+  const onToggle = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const onSelect = (
+    _event: React.MouseEvent<Element, MouseEvent> | undefined,
+    value: string | number | undefined,
+  ) => {
+    if (value && value !== 'no results') {
+      setSelected(value as string);
+    }
+    const [namespace, tempoStackName] = String(value).split(' / ');
```

Comment:
Is there a better way to get this data from an object rather than splitting a string?